util/predicates/generic_predicates.go

```diff
@@ -157,6 +177,12 @@ func ResourceNotPaused(logger logr.Logger) predicate.Funcs {
 	}
 }
 
+// ResourceNotPausedAndHasLabel returns a predicate that returns true only if the
+// ResourceNotPaused and ResourceHasLabel predicates return true.
+func ResourceNotPausedAndHasLabel(logger logr.Logger, labelValue string) predicate.Funcs {
```

Comment:
Could this be called `ResourceWatched` or `ResourceNotIgnored` maybe? It's hard to parse the meaning at a glance with the long name.

Reply:
I followed their precedent here - there were multiple functions like this all following this naming scheme. Currently `All` is completely unused but that's why I chose the name.

Reply:
I see. That works then.